Tools/TestWebKitAPI/Configurations/TestWebKitAPIApp.xcconfig

```diff
@@ -24,7 +24,7 @@
 #include "TestWebKitAPIBase.xcconfig"
 
 PRODUCT_NAME = TestWebKitAPI;
-PRODUCT_BUNDLE_IDENTIFIER = com.apple.WebKit.TestWebKitAPIApp;
+PRODUCT_BUNDLE_IDENTIFIER = org.webkit.$(PRODUCT_NAME:rfc1034identifier);
```

Comment:
This drops "App" from the identifier, is that intentional?

Reply:
It was intentional, since using `org.webkit` was enough to make this bundle ID distinct from the command-line tool.